src/fsharp/fsiAnyCpu/FsiAnyCPU.fsproj

```diff
@@ -11,10 +11,9 @@
     <ProjectGuid>{8b3e283d-b5fe-4055-9d80-7e3a32f3967b}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <NoWarn>$(NoWarn);62</NoWarn>
-    <AssemblyName>FsiAnyCPU</AssemblyName>
+    <AssemblyName>fsiAnyCPU</AssemblyName>
```

Comment:
Are you sure this is the right capitalization?
Assembly Names guidance is Pascal casing not camel casing?  Although they are in fact case insensitive in the runtime.

If you want to change the output filename on disk to camel case use the TargetFilename property.